server/rooms.ts

```diff
@@ -1420,7 +1420,13 @@ export class GlobalRoomState {
 		let privateCrashMessage = null;
 
 		const upperStaffRoom = Rooms.get('upperstaff');
-		if (stack.includes("private")) {
+		let hasPrivateTerm = stack.includes('private');
+
+		const privateTerms = (Config.privatecrashterms ? [...Config.privatecrashterms] : []).map(item => new RegExp(item, "i"));
+		for (const term of privateTerms) {
```

Comment:
```suggestion
		for (const term of (Config.privatecrashterms || [])) {
```

I'm not sure why you're converting `Config.privatecrashterms` to an array. What are you anticipating it being?